src/runner/config/defaultconfig_test.go

```diff
@@ -0,0 +1,79 @@
+package config
+
+import (
+	"fmt"
+	"net/url"
+	"testing"
+
+	"github.com/Arriven/db1000n/src/utils/ota"
+)
+
+//nolint: paralleltest // No need to test this in parallel :p
```

Comment:
please avoid using nolint at all costs. For now it is used only when the functionality of the app actually requires it (`InsecureSkipVerify: true` and cryptographic random)

Reply:
So, I would rather re-configure the linter that gives the warning. This is just a unit test that validates 10 lines of code - no need to add any kind of parallel testing here

Reply:
I feel like parallel testing is used here in order to speed up the testing process

Reply:
The problem here is that's unique case where a global variable is in the place (`ota.Version`) and running these test in parallel would lead to the problem of its concurrent modification.